shell/platform/android/test/io/flutter/embedding/engine/systemchannels/PlatformChannelTest.java

```diff
@@ -50,12 +52,12 @@ public void setSystemExclusionRectsSendsSuccessMessageToFramework() throws JSONE
         Rect gestureRect = new Rect(left, top, right, bottom);
         expectedDecodedRects.add(gestureRect);
 
-        MethodCall callSystemGestureExclusionRects = new MethodCall(
+        MethodCall callSetSystemGestureExclusionRects = new MethodCall(
             "SystemGestures.setSystemGestureExclusionRects",
             inputRects
         );
 
-        platformChannel.parsingMethodCallHandler.onMethodCall(callSystemGestureExclusionRects, resultsMock);
+        platformChannel.parsingMethodCallHandler.onMethodCall(callSetSystemGestureExclusionRects, resultsMock);
```

Comment:
I'm not sure how these tests overlap with the other PR, but can we separate the behavior under test from the verification? If it's already done in the other PR, then no worries.

Reply:
I think I mistakenly made this variable name update to this PR instead of [the more relevant one](https://github.com/flutter/engine/pull/11804). Should I move it over there instead? The other PR addresses separating the behavior under test from verification.

Reply:
Either place is fine as long as the right change is made :)